tests/test_lioness.py

```diff
@@ -65,13 +67,32 @@ def test_lioness():
     #pd.testing.assert_frame_equal(rdf, pydf, rtol=5e-1, atol= 99e-2,  check_exact=False, check_names = False, check_column_type = False)
     np.allclose(rdf.iloc[:,2].values, pydf.iloc[:,2].values)
 
+    # Check the same results for running only the first two samples
+    lioness_obj = Lioness(panda_obj,save_dir = "lioness_output", start=1, end = 2, save_single=True)
+    lioness_obj.save_lioness_results(output_results_npy_2)
+
+    # First, check that all save results are the same
+    res_npy2 = np.load(output_results_npy_2)
+    np.allclose(res_npy[:,:2], res_npy2)
+
+
+    # Check the same results for running only on sample 0 and 2
+    lioness_obj = Lioness(panda_obj,save_dir = "lioness_output", start=1, end = 4, subset_numbers='0,1,2', subset_names = '',save_single=True)
```

Comment:
why is subst_numbers a string rather than a numpy vector?

Reply:
good point, old habits

Reply:
Cool. Which one takes precedence when both START/END and subset_numbers are given?

Reply:
Ok, now it takes a list of integers of sample names as input. The precedence is given to the subsets (I have updated the documentation). If no subset is specified, then start and end are used (by default we always have start=1 and end=None).